lib/mix/lib/mix/scm/git.ex

```diff
@@ -307,4 +310,11 @@ defmodule Mix.SCM.Git do
     {int, _} = Integer.parse(string)
     int
   end
+
+  defp cmd_opts(opts \\ []) do
```

Comment:
I think we should document why this function exists and why we are calling File.cwd!.